.gitignore

```diff
@@ -8,3 +8,5 @@ target
 
 # Key pairs generated by the NEAR shell
 neardev
+
+.idea
```

Comment:
Yeah, @chadoh suggested to use global `.gitignore` since it's not related to the project.

Reply:
I find this suboptimal, as pretty much you are saying that every single developer in the world should have this configured?... Or be very careful about their adding code?

Reply:
Yes. It's correct. You add this once for your machine and then don't need to pollute gitignore of every project: https://docs.github.com/en/github/using-git/ignoring-files

The reason is if I don't use `clion` or other `.idea` IDE, then it's not needed for me in .gitignore

Reply:
this seems like a really weird discussion... having extra 5 bytes in the gitignore doesn't cost much and saves globally hours of work seems like a worthy exchange.

Reply:
I've gotten yelled at when I've added `.DS_Store` to the `.gitignore` of other open source projects, which made me think it's a bit of a standard practice to avoid adding every env- and editor-specific file to every project's `.gitignore`